Classes/Controller/Backend/ApplicationController.php

```diff
@@ -82,7 +82,7 @@ public function buildMenu()
         foreach ($this->menuUrls as $url) {
             //If extbase_acl is loaded, reduce menu urls to the ones actually allowed
             if (ExtensionManagementUtility::isLoaded("extbase_acl")) {
-                if (!\Pagemachine\ExtbaseAcl\Manager\ActionAccessManager::getInstance()->isActionAllowed(__CLASS__, $url['action'])) {
+                if (!\Pagemachine\ExtbaseAcl\Manager\ActionAccessManager::getInstance()->isActionAllowed(get_class($this), $url['action'])) {
```

Comment:
You can also use `static::class` for late static binding here.

Reply:
Changed, thanks.